src/Components/RecsListTable/index.js

```diff
@@ -296,6 +333,11 @@ const RecsListTable = () => {
     },
   ];
 
+  const onSort = (_e, index, direction) => {
+    dispatch(SortRecsTableIndex(index));
```

Comment:
A nitpick: as you don't change these index and direction values separately anywhere in the code, it could be better to keep only one action called `updatePagination` or update the pagination values using existing `updateFilters` like

```suggestion
    dispatch(
      updateFilters({ ...filters, sortIndex: index, sortDirection: direction })
    );
```

Reply:
I think I tried to do it again and had some issues with the fact that data sent to the state.recsListState.sortDirection was the same index as in the state.recsListState.sortIndex and not "desc"/"asc" strings. 

Let me try to implement that again